src/Zenaton/v1/Client.php

```diff
@@ -249,6 +252,146 @@ public function resumeWorkflow($workflowName, $customId)
         $this->updateInstance($workflowName, $customId, self::WORKFLOW_RUN);
     }
 
+    /**
+     * Schedule a task instance.
+     *
+     * @param string $cron
+     *
+     * @throws ApiException
+     *
+     * @return Schedule
+     */
+    public function scheduleTask(TaskInterface $task, $cron)
+    {
+        $name = \get_class($task);
+
+        $mutation = <<<'MUTATION'
```

Comment:
Why do you need the `'` around `MUTATION` here?

Reply:
If I don't add the `'`, PHP (💩) will try to interpolate variables inside the string. And since in GraphQL parameters starts with a `$`, which is the character for variables in PHP, it will try to interpolate the `$input` as a variable and will emit an error of undefined variable `$input`.

Reply:
Ah right. Tricky!